### PR TITLE
refactor(firebase): integrate cerebral-provider-firebase in monorepo

### PR DESCRIPTION
**Pull request by henri-hulski**

### CODE REVIEW

File: packages/cerebral-provider-firebase/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "cerebral-provider-firebase",
+  "version": "1.0.0-alpha.1",
+  "description": "Firebase provider for Cerebral",
+  "main": "lib/index.js",
+  "scripts": {
+    "test": "npm test",
```
Comment (Guria) [inline packages/cerebral-provider-firebase/package.json]: seems strange here, let's just remove it for now

Comment (henri-hulski) [inline packages/cerebral-provider-firebase/package.json]: Right. Removed.

**Comment by henri-hulski** [comment]
When running npm run build I get errors like:
```
> cerebral-provider-firebase@1.0.0-alpha.1 build /home/pacha/web/cerebral/cerebral/packages/cerebral-provid
er-firebase
> BABEL_ENV=production ../../node_modules/.bin/babel src/ --out-dir=lib/ -s

src/createOnChildAdded.js -> lib/createOnChildAdded.js
SyntaxError: src/createOnChildChanged.js: Unexpected token (17:10)
  15 |           key: data.key,
  16 |           value: data.val(),
> 17 |           ...options.payload
     |           ^

  18 |         })
  19 |       }
  20 |     )
```
Similar in other files.

**Comment by Guria** [comment]
@henri-hulski `.babelrc` file ?

**Comment by henri-hulski** [comment]
Added now, but doesn't help.

**Comment by henri-hulski** [comment]
@Guria On Travis we get the same error.

**Comment by Guria** [comment]
hm strange. ok will check this out.

**Comment by Guria** [comment]
Of course. It requires stage-3 at least
http://babeljs.io/docs/plugins/preset-stage-3/

**Comment by henri-hulski** [comment]
@Guria Yeah! That fixed it.

**Pull request merged**